modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts

```diff
@@ -29,14 +31,16 @@ export class JournalEditArticlePage {
 		this.changesSavedIndicator = page.locator(
 			'#_com_liferay_journal_web_portlet_JournalPortlet_changesSavedIndicator'
 		);
+		this.friendlyURLInput = page.locator('#_com_liferay_journal_web_portlet_JournalPortlet_friendlyURL');
+		this.friendlyUrlToggle = page.locator('#friendlyUrlToggle');
 		this.journalPage = new JournalPage(page);
 		this.propertiesTab = page.getByRole('tab', {name: 'Properties'});
-		this.publishButton = page.getByRole('button', {name: 'Publish'});
+		this.publishButton = page.locator('#_com_liferay_journal_web_portlet_JournalPortlet_publishButton');
```

Comment:
It is better to look by roles (`getByRole()`) or placeholders(`getByPlaceholder()` for the `titleInput`), so we are considering at the same time accesibility issues. 
Is there a real reason to change `publishButton` and `titleInput` with locators? 

If not, check also it friendlyURLInput/Toggle can be get by accesible getters (use` npm run playwright codegen http://localhost:8080 `to check)

Reply:
Thanks @ambrinchaudhary ,
The reason I change is that I changed the language before to french and "Publish" etc names are changing and I wanted to them more language friendly for the later usage.
I will check if I can find somthing for getByRole or getByPlaceholder
Thanks

Reply:
Got it, then is it ok with the id.